tests/workflows/README.md

```diff
@@ -0,0 +1,4 @@
+# Workflows for tests against compound databases
+
+## Invariance test with the [ChEMBL database](https://www.ebi.ac.uk/chembl/)
+`snakemake -jall --snakefile Snakefile_ChEMBL test_invariance`
```

Comment:
Do we also need instructions regarding the required input (i.e., sqlite file)?

Reply:
ChEMBL's sqlite file is downloaded via a rule. If the file already exists, the download will be skipped. I think it's not worth mentioning.

Reply:
Fair enough, I should've looked at the snakefile more closely. In the long run we should add docs on how to run the database validations, once we have a way to test the CSD as well.